package.json

```diff
@@ -80,7 +80,7 @@
     "css-loader": "^6.7.1",
     "dateformat": "^5.0.3",
     "dotenv": "^16.0.1",
-    "electron": "^v27.1.2",
+    "electron": "^v28.1.0",
```

Comment:
Why not update to latest version [v28.1.4](https://releases.electronjs.org/release/v28.1.4)

Reply:
ok, let me check it